src/entities/User.java

```diff
@@ -27,6 +35,19 @@ public User(String nameUser, String surnameUser, int age, LocalDate birthday, St
         this.password = password;
 
     }
+
+
+    public void registerUser(String nameUser, String surnameUser, Integer age, Date birthday, Brand sex, String email, String password) throws SQLException {
```

Comment:
non capisco dove avviene la connessione al db in questo metodo.

anche di questo devi fare il test.

Reply:
Si ho dimenticato di aggiungerlo, va bene per fare il test